server.js

```diff
@@ -54,6 +54,13 @@ router.get(['/', '/index.html'], (req, res) => {
   });
 });
 
+router.get('/volunteer', (req, res) => {
+  res.render('volunteer', {
+    ogTitle: res.locals.banana.i18n('ftm-index-og-title'),
+    ogUrl: `http://${req.hostname}${req.originalUrl}`
```

Comment:
can you add ogDescription too?

Reply:
Maybe also change `http` to `${req.protocol}` so it's `https` in production and `http` in dev.